src/consts.js

```diff
@@ -93,21 +93,21 @@ export const activeTechnologies = [
         body: 'Install, register, and manage Red Hat OpenShift® 4 clusters.'
     },
     {
-        id: 'Hybrid',
-        url: 'hybrid',
+        id: 'Cost Management',
+        url: 'cost-management',
         baseApp: '/cost-management',
-        entitlement: 'hybrid_cloud',
+        entitlement: 'cost_management',
```

Comment:
@ryelo This isn't the same as the response we get back from `entitlements` right?

Reply:
I was in the process of making the changes in other places. Please check the PR description now!

Reply:
Got it! Thanks a lot!